app/routes/explore.js

```diff
@@ -65,6 +65,18 @@ export default Route.extend({
       });
     }
 
+    if (params.cfs) {
+      let cfsVal = false;
+      if (params.cfs === 'open') {
```

Comment:
this entire block can be reduced to a single line:
```js
let cfsVal = (params.cfs === 'open' ? true: false);

Reply:
@CosmicCoder96 Requested changes completed.